### PR TITLE
feature: add legend to big map view

### PR DESCRIPTION
**Pull request by tysonanderson**
Initial version of map legend with thresholds for floodnet. Mobile view disabled but forthcoming.

### CODE REVIEW

**Comment by jlewallen** [comment]
Oooh nice. Is the plan to also pull the thresholds from the visible sensor's meta data?

File: portal/src/views/projects/ProjectBigMap.vue [hunk 1]
```diff
@@ -111,6 +121,17 @@ export default Vue.extend({
         exploreContext(): ExploreContext {
             return new ExploreContext(this.project.id);
         },
+        levels(): object[]{
```
Comment (jlewallen) [inline portal/src/views/projects/ProjectBigMap.vue:124]: I would return from inside the try and then return an empty list from the catch or outside the try/catch.

**Comment by jlewallen** [comment]
Much better, thanks. The only thing that concerns me is the assumptions made on indexing into those configurations for the labels. We can address that when we return to this to make this legend a "per project" setting, which is something Jer and I discussed.

**Comment by tysonanderson** [comment]
Agreed. It's a deep query reliant on data being available in several places which is why I put the try/catch block but it's definitely a stop gap until we revisit to scale up to all projects.

**Pull request merged**